scss/_buttons.scss

```diff
@@ -23,7 +23,7 @@
     $btn-disabled-color
   );
 
-  border-color: currentColor !important;
+  border-color: currentcolor !important;
```

Comment:
https://www.w3.org/TR/css-color-4/#currentcolor-color

Reply:
This one i am nervous about as in the css-color-3 version of the link then currentColor is used....

Reply:
I also cannot find any visual changes in the bootstrap theme.